### DISCUSSION TITLE
SilentStudio on Macbook Pro M1 Pro works!

### DISCUSSION THREAD

**Discussion by MiroIsNot1337** [dirkschreib/SilentStudio#11] [General]
I am not sure if it's an intended use, but I purchased a Macbook Pro with an M1 Pro processor and migrated my data from my Mac Studio's time machine backup and I can confirm that SilentStudio works flawlessly on it. It displays the temperature and I can set fan speeds without problems. This post is mostly so that users who wonder about it know that SilentStudio seems to have at least some compatibility with Macbooks.

It was more by mistake than design, but I enjoy having control over the fan which for my taste kicks in way too late (after a minute or two of 95-105°C). I have created a fan curve that suits me better and I might refine it more. The RPM is a lot higher than on the Mac Studio (I think) with over 5000rpm max. I am not even sure anymore what the Studio's fan sounds like because it kicks in only rarely when I edit videos.

Thanks again for the great work, Dirk!

**Comment by dirkschreib** [comment]
Thanks for the information and the kind words. I don‘t have a MacBook Pro myself and can‘t test it. It all depends on Apple using the same SMC-Keys for the temperature sensors and fans.
